src/components/todoItem/index.js

```diff
@@ -1,37 +1,20 @@
 import React from "react"
 import PropTypes from "prop-types"
-import ImageButton from "components/buttons/imageButton"
-import "components/todoItem/todoItem.scss"
-
-import {
-  COMPLETE_BUTTON_ALT,
-  DELETE_BUTTON_ALT,
-  EDIT_BUTTON_ALT,
-  IMAGE_PATHS,
-} from "utils/constants"
+import TodoButtonsContainer from "components/todoButtonsContainer"
 import "components/todoItem/todoItem.scss"
 
 function TodoItem({ todo }) {
-
+  
   return (
-    <div className="item-card">
-      <h2>{todo.text}</h2>
-      <div className="item-card__content">
-        <p className="item-card__time"> Created At : {todo.createdAt}</p>
-        <div className="item-card__actions">
-          <ImageButton
-            icon={IMAGE_PATHS.CHECK}
-            buttonAltText={COMPLETE_BUTTON_ALT}
-          />
-          <ImageButton
-            icon={IMAGE_PATHS.EDIT}
-            buttonAltText={EDIT_BUTTON_ALT}
-          />
-          <ImageButton
-            icon={IMAGE_PATHS.DELETE}
-            buttonAltText={DELETE_BUTTON_ALT}
-          />
-        </div>
+    <div className="item_card">
+      <h2 className={todo.completed === true ? "completed" : ""}>
```

Comment:
```suggestion
    <div className="item_card">
      <h2 className={todo.completed === true ? "completed" : null}>
```

Reply:
Bhaia i removed this part since it is related to step-4. Thanks